homeassistant/components/cover/tahoma.py

```diff
@@ -66,13 +172,58 @@ def device_class(self):
             return 'window'
         return None
 
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        attr = {}
+        super_attr = super().device_state_attributes
+        if super_attr is not None:
+            attr.update(super_attr)
+
+        if self._closure is not None:
+            attr[ATTR_CLOSURE] = self._closure
+        if 'core:Memorized1PositionState' in self.tahoma_device.active_states:
+            attr[ATTR_MEM_POS] = self.tahoma_device.active_states[
+                'core:Memorized1PositionState']
+        if self._rssi_level is not None:
+            attr[ATTR_RSSI_LEVEL] = self._rssi_level
+        if 'core:OpenClosedState' in self.tahoma_device.active_states:
+            attr[ATTR_OPEN_CLOSE] = self.tahoma_device.active_states[
+                'core:OpenClosedState']
+        # Can be 'available', 'unavailable'
+        if 'core:StatusState' in self.tahoma_device.active_states:
+            attr[ATTR_STATUS] = self.tahoma_device.active_states[
+                'core:StatusState']
+        if self._lock_timer > 0:
+            attr[ATTR_LOCK_TIMER] = self._lock_timer
+        if self._lock_start_ts is not None:
+            attr[ATTR_LOCK_START_TS] = self._lock_start_ts.isoformat()
+        if self._lock_end_ts is not None:
+            attr[ATTR_LOCK_END_TS] = self._lock_end_ts.isoformat()
+        if self._lock_level is not None:
+            attr[ATTR_LOCK_LEVEL] = self._lock_level
```

Comment:
I am not sure, it is an attribute provided by the Tahoma API. My best guess is, that it describes the level of the lock, eg. the strength of the wind. I usually see 'comfortLevel1' with my covers in case of wind.
In the code I added all possible values:
Can be 'comfortLevel1', 'comfortLevel2', 'comfortLevel3', 'comfortLevel4', 'environmentProtection', 'humanProtection', 'userLevel1', 'userLevel2'